tests/saw/spec/handshake/handshake_io_lowlevel.saw

```diff
@@ -313,12 +322,15 @@ let s2n_connection_get_client_auth_type_spec = do{
 // by conn_set_handshake_type (low-level model function)
 let s2n_conn_set_handshake_type_spec chosen_psk_null = do {
     (pconn, conn) <- setup_connection_common chosen_psk_null;
-    // we assume that the handshake struct denotes a valid handshake state
+    // We assume that the handshake struct denotes a valid handshake state
     // (e.g. it will not index out of bounds in the state transition array
     // "handshakes")
+    // In addition, it satisfies that if actual_protocol_version is set to
```

Comment:
It's probably better to put this description near the definition of `valid_connection`. If you put it here, then it won't be maintained when `valid_connection` is modified.